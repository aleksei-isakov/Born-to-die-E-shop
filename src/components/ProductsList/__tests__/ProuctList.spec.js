import { mount } from '@vue/test-utils';
import ProductList from '@/components/ProductsList/ProductsList.vue';

describe('ProductList', () => {
  it('renders correctly', () => {
    const wrapper = mount(ProductList);

    expect(wrapper.element).toMatchSnapshot();
  });
});
