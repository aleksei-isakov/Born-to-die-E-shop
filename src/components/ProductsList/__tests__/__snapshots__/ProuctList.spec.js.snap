// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductList renders correctly 1`] = `
<div
  class="container"
>
  <div
    class="products-list"
  >
    <div
      class="recently-added"
    >
      Recently added
    </div>
     
    <span
      class="found-products"
    >
      Found: 5
    </span>
     
    <ul
      class="products-list__items"
    >
      <li
        class="product-item"
      >
        <img
          alt="product image"
          class="product-item__image"
          src=""
        />
         
        <div
          class="product-description"
        >
          <h2
            class="product-description__title"
          >
            Soft Soap
          </h2>
           
          <p
            class="product-description__field"
          >
            HOME
          </p>
           
          <span>
            Created: 2.2.2022
          </span>
           
          <br />
           
          <span>
            Updated: 2.2.2022
          </span>
        </div>
         
        <div
          class="product-item__price-container"
        >
          <div
            class="product-price"
          >
            800.7 $
          </div>
           
          <button
            class="base-button md-button md-raised md-primary md-theme-default add-btn"
          >
            
      + ADD TO CART
    
          </button>
        </div>
      </li>
      <li
        class="product-item"
      >
        <img
          alt="product image"
          class="product-item__image"
          src=""
        />
         
        <div
          class="product-description"
        >
          <h2
            class="product-description__title"
          >
            Wooden Chair
          </h2>
           
          <p
            class="product-description__field"
          >
            HOME
          </p>
           
          <span>
            Created: 2.2.2022
          </span>
           
          <br />
           
          <span>
            Updated: 2.2.2022
          </span>
        </div>
         
        <div
          class="product-item__price-container"
        >
          <div
            class="product-price"
          >
            800.0 $
          </div>
           
          <button
            class="base-button md-button md-raised md-primary md-theme-default add-btn"
          >
            
      + ADD TO CART
    
          </button>
        </div>
      </li>
      <li
        class="product-item"
      >
        <img
          alt="product image"
          class="product-item__image"
          src=""
        />
         
        <div
          class="product-description"
        >
          <h2
            class="product-description__title"
          >
            Plastic Salad
          </h2>
           
          <p
            class="product-description__field"
          >
            GAMES
          </p>
           
          <span>
            Created: 2.2.2022
          </span>
           
          <br />
           
          <span>
            Updated: 2.2.2022
          </span>
        </div>
         
        <div
          class="product-item__price-container"
        >
          <div
            class="product-price"
          >
            800.0 $
          </div>
           
          <button
            class="base-button md-button md-raised md-primary md-theme-default add-btn"
          >
            
      + ADD TO CART
    
          </button>
        </div>
      </li>
      <li
        class="product-item"
      >
        <img
          alt="product image"
          class="product-item__image"
          src=""
        />
         
        <div
          class="product-description"
        >
          <h2
            class="product-description__title"
          >
            Plastic Salad
          </h2>
           
          <p
            class="product-description__field"
          >
            GAMES
          </p>
           
          <span>
            Created: 2.2.2022
          </span>
           
          <br />
           
          <span>
            Updated: 2.2.2022
          </span>
        </div>
         
        <div
          class="product-item__price-container"
        >
          <div
            class="product-price"
          >
            800.0 $
          </div>
           
          <button
            class="base-button md-button md-raised md-primary md-theme-default add-btn"
          >
            
      + ADD TO CART
    
          </button>
        </div>
      </li>
      <li
        class="product-item"
      >
        <img
          alt="product image"
          class="product-item__image"
          src=""
        />
         
        <div
          class="product-description"
        >
          <h2
            class="product-description__title"
          >
            Plastic Salad
          </h2>
           
          <p
            class="product-description__field"
          >
            GAMES
          </p>
           
          <span>
            Created: 2.2.2022
          </span>
           
          <br />
           
          <span>
            Updated: 2.2.2022
          </span>
        </div>
         
        <div
          class="product-item__price-container"
        >
          <div
            class="product-price"
          >
            800.0 $
          </div>
           
          <button
            class="base-button md-button md-raised md-primary md-theme-default add-btn"
          >
            
      + ADD TO CART
    
          </button>
        </div>
      </li>
    </ul>
  </div>
</div>
`;
